### ISSUE TITLE
Requesting symbol sometimes throws exception and returns null

### ISSUE THREAD

**Issue by y2kbowen** [missioncommand/old-mil-sym-java#38]
Sometimes a request for a symbol causes an exception in the rendering service and returns null to the requester. I have attached pictures from a fiddler trace along with a picture of the corresponding exception in the renderer service console.  

You sometimes have to try many tries from fiddler to get it to happen. If you would like me to post the fiddler session to help I can do that.
![fiddler_firstcall](https://cloud.githubusercontent.com/assets/4991352/21656311/3e992a60-d283-11e6-8f21-cba3e43005fb.jpg)
![fiddler_secondcall](https://cloud.githubusercontent.com/assets/4991352/21656313/3e9c91be-d283-11e6-84dc-4cfa6c03b59f.jpg)
![multipointserver_console_exception](https://cloud.githubusercontent.com/assets/4991352/21656312/3e99daf0-d283-11e6-92b8-da4e014b5d81.JPG)

**Comment by michael-spinelli** [comment]
In your testing, can this happen at any time or only when many requests are being made to the server at the same time?

From the screenshot it looks like you were making repeated calls with the same parameters, correct?

**Comment by y2kbowen** [comment]
In your testing, can this happen at any time or only when many requests are being made to the server at the same time?
   For testing I created a localhost server and made requests from fiddler one at a time until the   exception was thrown. No other requests were being made to the service.

From the screenshot it looks like you were making repeated calls with the same parameters, correct?
  Yes, I was using Fiddler to send the same request over and over until the error occurred.   I did this because I noticed in my actual application an exception being logged every now and then and wondered what was happening.
